checklists/newHire.json

```diff
@@ -111,7 +112,7 @@
 	},
 	"attendAllHands": {
 		"displayName": "Attend the weekly all-hands",
-		"description": "Join our weekly meeting <a href=\"https://handbook.18f.gov/distributed/#heading-2\">online</a>, or ask your in-office buddy for details.",
+		"description": “The all-hands meeting takes place at 12:30pm ET every Tuesday.  Leah Gitter will add you to the calendar invite.”,
```

Comment:
@keithrwilson: I think the special quotation marks (`“` and `”`) at the beginning and end of this description are making the JSON invalid. Could you replace them with regular quotation marks (`"`)?